examples/natmod/features2/Makefile

```diff
@@ -10,5 +10,8 @@ SRC = main.c prod.c test.py
 # Architecture to build for (x86, x64, armv7m, xtensa, xtensawin)
 ARCH = x64
 
+# Link with libm.a and libgcc.a from the toolchain
+MICROPY_LINK_RUNTIME = 1
```

Comment:
The other Make variables like `ARCH` are simple, short names, so how about calling this just `LINK_RUNTIME` to match them?